### PR TITLE
FC-1195 fix doc typos

### PR DESCRIPTION
**Pull request by dpetran**

### CODE REVIEW

File: src/content/1.0.0/docs/start/installation.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ Property | Options | Description
...
 `fdb-group-timeout` | `int` | Tx group's internal communication timeout threshold. Will initiate a leader election between this value and 2x this value if the leader hasn't been heard from. Specify as number of milliseconds, or can use units as well such as 1000ms or 1s. Assuming your tx-group network is local, 1000-3000 ms is a good range. Adjust as needed to avoid unintended leader elections.
-`fdb-group-hearbeat` | `int` | Tx group leader will send out a heartbeat at this interval. By default, will be 1/2 of fdb-group-timeout. This can never be less than fdb-group-timeout, and ideally should be 1/3 to 1/2 of that value. A number in milliseconds can be provided, or can be used with units such as 1000ms or 1s.
+`fdb-group-hearbeat` | `int` | Tx group leader will send out a heartbeat at this interval. By default, will be 1/2 of fdb-group-timeout. This can never be more than fdb-group-timeout, and ideally should be 1/3 to 1/2 of that value. A number in milliseconds can be provided, or can be used with units such as 1000ms or 1s.
```
Comment (dpetran) [inline src/content/1.0.0/docs/start/installation.md:174]: This is a semantic as opposed to a grammar/spelling change, but I think it's correct.

File: src/content/1.0.0/guides/analytical_queries/full-text-search.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ Now, we can issue a full text query against any predicate or collection by using
 {
     "select": "?person",
     "where": [
-        ["?person", "fullText:person/handle", "jdoe"]
+        ["?person", "fullText:person/handle", "doe"]
```
Comment (dpetran) [inline src/content/1.0.0/guides/analytical_queries/full-text-search.md:96]: This is a semantic change as well, but I think it reflects the text above the example.

**Comment by dpetran** [comment]
These are just for the 1.0.0 branch, I can also backport them to previous versions.

File: src/content/1.0.0/guides/analytical_queries/full-text-search.md (hunk 2 above)
Comment (Tr3yb0) [reply src/content/1.0.0/guides/analytical_queries/full-text-search.md:96]: this is the person/handle in the data set and is used multiple times. it should stay as jdoe.

File: src/content/1.0.0/docs/start/installation.md (hunk 1 above)
Comment (Tr3yb0) [reply src/content/1.0.0/docs/start/installation.md:174]: Can you update the parameter to be `fdb-group-heartbeat` as well?

**Comment by Tr3yb0** [comment]
While the updates in this PR are needed, I would prefer to batch them more because there will need to be a _lot_ of deployments, which are currently manual. If possible, I would rather push multiple commits for a set of pages and then create a PR off of that as opposed to doing a PR per page.

File: src/content/1.0.0/docs/start/intro.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ The Docs section is a basic technical overview of key topics, including:
 - Ledger Set-up
 - Infrastructure
 
-To view the documentation examples in different programming languages, you can toggle the language options in the sidebar on the top right. 
+To view the documentation examples in different programming languages, you can toggle the language options in the left sidebar on the top right.
```
Comment (Tr3yb0) [inline src/content/1.0.0/docs/start/intro.md:12]: i think you can remove "on the top right"

**Review by Tr3yb0 — COMMENTED** [review]
Added a few comments

File: src/content/1.0.0/guides/analytical_queries/full-text-search.md (hunk 2 above)
Comment (dpetran) [reply src/content/1.0.0/guides/analytical_queries/full-text-search.md:96]: There's something weird going on here, I tried actually running the transactions and queries specified here and I'm not getting the same results. I've filed FC-1196 to track that.

**Review by Tr3yb0 — APPROVED** [review]

**Comment by Tr3yb0** [comment]
approved, thanks for picking up the typos and fixing the full text docs!

**Pull request merged**